### PR TITLE
1374 container criteria new condition in children

### LINKED ISSUE #1374: Container criteria: new condition "In children"
image

We have the container condition `In parent` which is very useful to get certain parameters located in the parent container of the current container.

It would be also very useful to have the `In children` condition, with the behavior similar to the `In container` condition but **excluding** the container itself.

Use case - a parameter `Volume` of the container `Organism` should sum up all `Volume` parameters of the childer of `Organism`. Right now, we require an additional tag assigned to all children containers of "Organism".

### PR DESCRIPTION
**Pull request by msevestre**
Fixes #1374 

# Description
This is just the plumbing. I need to wait for https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/pull/2360/files to be merged

## Type of change

Please mark relevant options with an `x` in the brackets.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires documentation changes (link at least one [user](https://github.com/Open-Systems-Pharmacology/docs) or [developer](https://github.com/Open-Systems-Pharmacology/developer-docs) documentation issue):
- [ ] Algorithm update - updates algorithm documentation/questions/answers etc.
- [ ] Other (please describe):

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Integration tests
- [ ] Unit tests
- [ ] Manual tests 
- [ ] No tests required

# Reviewer checklist

Mark everything that needs to be checked before merging the PR.

- [ ] Check if the code is well documented
- [ ] Check if the behavior is what is expected
- [ ] Check if the code is well tested
- [ ] Check if the code is readable and well formatted
- [ ] Additional checks (document below if any)
- [ ] Check if documentation update issue(s) are created if the option `This change requires a documentation update` above is selected

# Screenshots (if appropriate):

# Questions (if appropriate):

### CODE REVIEW

File: src/MoBi.Presentation/Tasks/SimulationCommitTask.cs [hunk 1]
```diff
@@ -113,11 +111,9 @@ private IEnumerable<IMoBiCommand> updateParameterValuesFromSimulationChanges(Mod
 
       /// <summary>
       ///    Creates two new macro commands that have synchronized the initial conditions building blocks from the template
-      ///    module and simulation module respectively
```
Comment (msevestre) [inline src/MoBi.Presentation/Tasks/SimulationCommitTask.cs:116]: unrelated to PR but warnings in console because of ref to simulation

**Review by msevestre — COMMENTED** [review]
seems to work. Right now references local nuget package. Will be merged once we merge core back in

File: src/MoBi.Assets/MoBi.Assets.csproj [hunk 2]
```diff
@@ -26,9 +26,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OSPSuite.Assets" Version="12.0.335" />
-    <PackageReference Include="OSPSuite.Assets.Images" Version="12.0.335" />
-    <PackageReference Include="OSPSuite.Core" Version="12.0.335" />
+    <PackageReference Include="OSPSuite.Assets" Version="12.0.0-OGKUq" />
+    <PackageReference Include="OSPSuite.Assets.Images" Version="12.0.0-OGKUq" />
+    <PackageReference Include="OSPSuite.Core" Version="12.0.0-OGKUq" />
```
Comment (Yuri05) [inline src/MoBi.Assets/MoBi.Assets.csproj:31]: core with the "in children" extension was merged, so now we can replace this with that version

File: src/MoBi.Core/Commands/AddTagConditionCommand.cs [hunk 3]
```diff
@@ -68,6 +66,25 @@ protected override ICommand<IMoBiContext> GetInverseCommand(IMoBiContext context
       }
    }
 
+   public class AddInChildrenConditionCommand<T> : AddTagConditionCommandBase<T> where T : class, IObjectBase
+   {
+      public AddInChildrenConditionCommand(TagConditionCommandParameters<T> tagConditionCommandParameters)
+         : base(string.Empty, tagConditionCommandParameters)
+      {
+         ObjectType = AppConstants.Commands.InParentCondition;
```
Comment (Yuri05) [inline src/MoBi.Core/Commands/AddTagConditionCommand.cs:74]: `InParentCondition `??

File: src/MoBi.Core/Commands/RemoveTagConditionCommand.cs [hunk 4]
```diff
@@ -70,6 +70,27 @@ protected override void RemoveTagCondition(DescriptorCriteria descriptorCriteria
       }
    }
 
+   public class RemoveInChildrenConditionCommand<T> : RemoveTagConditionCommandBase<T> where T : class, IObjectBase
+   {
+      public RemoveInChildrenConditionCommand(TagConditionCommandParameters<T> tagConditionCommandParameters)
+         : base(string.Empty, tagConditionCommandParameters)
+      {
+         ObjectType = AppConstants.Commands.InParentCondition;
```
Comment (Yuri05) [inline src/MoBi.Core/Commands/RemoveTagConditionCommand.cs:78]: `InParentCondition `?

**Pull request merged**